UIPS/UIP-0128.md

```diff
@@ -16,6 +16,10 @@ read from the namespace without using dotket (`.^`) or its virtual
 Nock operator 12. Once we remove Nock 12, _all_ code in the system
 can take advantage of persistent memoization.
 
+## Status
+
+This UIP is marked stagnant, ~mastyr-bottec implemented a version of this which can be found [here](https://github.com/urbit/urbit/pull/6842).
```

Comment:
Please add `~fodwyt-ragful` here so it says "~fodwyt-ragful and ~mastyr-bottec implemented..."